### PR TITLE
Downloads: Add flatpak installation for Linux

### PR DESCRIPTION
**Pull request by manuq**
Under Linux section.

Note: the links will work only after the flatpak pull-request is merged
in the mypaint repository.

### CODE REVIEW

File: downloads.md [hunk 1]
```diff
@@ -32,6 +32,27 @@ the “master” branch on GitHub.
...
+Ubuntu, elementaryOS, Arch, openSuSE, and many more.
+
+[Click to install](https://raw.githubusercontent.com/mypaint/mypaint/master/flatpak/mypaint-stable.flatpakref)
+
+Click to install is not yet availble in all distributions.  If you are
+lucky it will open your Software application.  Otherwise you can use
+the command line:
+
+```
+flatpak install --from https://raw.githubusercontent.com/mypaint/mypaint/master/flatpak/mypaint-stable.flatpakref
```
Comment (manuq) [inline downloads.md:46]: As said in the commit message, the link won't work until my flatpak PR is merged in mypaint repo. But you can give it a try from my branch like this:

```
flatpak install https://raw.githubusercontent.com/manuq/mypaint/flatpak/flatpak/mypaint-stable.flatpakref
flatpak run org.mypaint.MyPaint
```

File: downloads.md [hunk 2]
```diff
@@ -32,6 +32,27 @@ the “master” branch on GitHub.
...
+```
+
+After installing the Flatpak, the applications will show up in your
+system, but you can also launch it from the commandline:
+
```
Comment (odysseywestra) [inline downloads.md]: You might also want to add that a they may have to log out and log back in for the Desktop launcher to show.

**Comment by odysseywestra** [comment]
Okay thanks for the PR. As soon as the PR for the flatpak package is merge, I'll merge this one.

File: downloads.md (hunk 2 above)
Comment (manuq) [reply downloads.md]: @odysseywestra under which desktop? In GNOME Shell after installing a flatpak the desktop displays the launcher without needing to log out. But yes if there is at least one desktop not displaying the launcher automatically, is worth mentioning.

Comment (odysseywestra) [reply downloads.md]: I'm using Gnome Shell on Antegros. Acording to the [Arch Wiki](https://wiki.archlinux.org/index.php/Flatpak) it was warned that we may have to restart the session for the launcher to show.

**Pull request merged**